packages/cloud2edge/Chart.yaml

```diff
@@ -11,8 +11,8 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 apiVersion: v1
-version: 0.3.0
-appVersion: 0.3.0
+version: 0.2.5
```

Comment:
the version number should be *increased*, not *decreased* ...

Reply:
this is an increase.
I had suggested 0.3.0 from 0.2.4 but @thjaeckle suggested that 0.2.5 is more appropriate.

Reply:
my bad, I only saw the diff to the previous commit ;-)

Reply:
Seeing that the changes here involve the upgrade from Hono 1.x to Hono 2.1, I think 0.3.0 would be more appropriate.

Reply:
updated to 0.3.0